### DISCUSSION TITLE
Error Highlighting

### DISCUSSION THREAD

**Discussion by puurrplecat** [helix-editor/helix#4057] [Q&A] [answered]
Hi,
It's my first time using Helix, and I've noticed that when my Rust-analyser highlights errors, it highlights the text fully (see image below), unlike other editors like VSCode where it just underlines. Is it possible to change this to an underline?
![image](https://user-images.githubusercontent.com/61997526/193376677-78e34fef-0fa3-4fd9-bb35-90aeeda664a5.png)

**Comment by kirawi** [comment, marked as answer]
It's defined by the theme you're using, e.g. https://github.com/helix-editor/helix/blob/60aa7d360777331753ae4735f8fd4ea96012d4fd/runtime/themes/dark_plus.toml#L95

**Reply by puurrplecat** [reply]
I see thanks :)
